packages/framework/ini-files/config-specs.ini

```diff
@@ -2300,6 +2320,7 @@ use USE-RDC|NO
 use USE-UVM|YES
 use USE-DEPRECATED|YES
 use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
+use PACKAGE-ENABLES|NO-EPETRA
```

Comment:
This one can stay as NO-PACKAGE-ENABLES, that won't turn anything on.  We only need the ALL replaced with ALL-NO-EPETRA